### PR TITLE
add /osg/ligo mapping (SOFTWARE-2762)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

**Comment by edquist** [comment]
@bbockelm, my main questions on this are:
- the `matchFQAN` / `acceptProxyWithoutFQAN` settings for the new `vomsUserGroup`
- the `accountingVoSubgroup` setting for the `groupToAccountMapping` (`osg` vs `ligo`)

hopefully they're set up right but let me know if you were expecting something different.

File: gums.config.template [hunk 1]
```diff
@@ -1609,6 +1623,14 @@
 			accountMappers='ilc'/>
 
 		<groupToAccountMapping
+			name='ligo'
+			description=''
+			accountingVoSubgroup='osg'
```
Comment (bbockelm) [inline gums.config.template]: Please change this and the next line to `ligo` & `LIGO`, respectively.

**Comment by edquist** [comment]
Interesting, ok, i guess the accounting VO doesn't have to be a real VO name then?

Anyway, updated the PR.

**Comment by bbockelm** [comment]
Correct - it's mostly for aesthetic purposes and not necessarily tied to VOMS.  (However, it's fairly rarely used...)

**Pull request merged**